DatadogInternal/Sources/NetworkInstrumentation/URLSession/URLSessionSwizzler.swift

```diff
@@ -22,6 +24,9 @@ internal class URLSessionSwizzler {
     internal private(set) static var dataTaskWithURL: DataTaskWithURL?
 
     static func bind() throws {
+        lock.lock()
+        defer { lock.unlock() }
+
         guard bindingsCount == 0 else {
             return bindingsCount += 1
```

Comment:
this statement looks so odd, the method doesn't return but still has a returning value.

Reply:
The operator and the `bind` both return `Void` :)